### PR TITLE
[Docs] Updates Code of Conduct

### LINKED ISSUE #1897: Fix Code of Conduct formatting
Either reference the [AWS Code of Conduct](https://aws.github.io/code-of-conduct) or fix the formatting and add attribution to the existing one.

* In particular, the meaning of the last paragraph is changed due to its heading being mixed into the examples of unacceptable behavior.

Perhaps a suitable one would be the default:
```
## Code of Conduct
This project has adopted the [Amazon Open Source Code of Conduct](https://aws.github.io/code-of-conduct).
For more information see the [Code of Conduct FAQ](https://aws.github.io/code-of-conduct-faq) or contact
opensource-codeofconduct@amazon.com with any additional questions or comments.
```

### PR DESCRIPTION
**Pull request by bandinib-amzn**
### Description
Updates code of conduct.
 
### Issues Resolved
#1897

Signed-off-by: Bandini Bhopi <bandinib@amazon.com>
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
    - [ ] `yarn test:jest`
    - [ ] `yarn test:jest_integration`
    - [ ] `yarn test:ftr`
- [ ] New functionality has been documented.
- [X] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,25 +1,4 @@
...
-* Public or private harassment of any kind;
-* Publishing private information, such as physical or electronic address, without permission;
-* Other conduct which could reasonably be considered inappropriate in a professional setting;
-* Advocating for or encouraging any of the above behaviors.
-* Enforcement and Reporting Code of Conduct Issues:
-
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported. [Contact us](mailto:opensource-codeofconduct@amazon.com). All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances.
\ No newline at end of file
+## Code of Conduct
+This project has adopted the [Amazon Open Source Code of Conduct](https://aws.github.io/code-of-conduct).
```
Comment (joshuarrrr) [inline CODE_OF_CONDUCT.md:2]: nit - should be blank line after headers

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,25 +1,4 @@
...
-* The use of sexualized language and unwelcome sexual attention or advances;
-* Public or private harassment of any kind;
-* Publishing private information, such as physical or electronic address, without permission;
-* Other conduct which could reasonably be considered inappropriate in a professional setting;
-* Advocating for or encouraging any of the above behaviors.
-* Enforcement and Reporting Code of Conduct Issues:
-
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported. [Contact us](mailto:opensource-codeofconduct@amazon.com). All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances.
\ No newline at end of file
+## Code of Conduct
```
Comment (joshuarrrr) [inline CODE_OF_CONDUCT.md:1]:
```suggestion
# Code of Conduct
```

**Review by joshuarrrr — APPROVED** [review]
Some formatting suggestions, but otherwise looks good!

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (AMoo-Miki) [reply CODE_OF_CONDUCT.md:2]: This should be in sync with [all other repos](https://github.com/search?q=org%3Aopensearch-project+https%3A%2F%2Faws.github.io%2Fcode-of-conduct-faq&type=code) OpenSearch has.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (AMoo-Miki) [reply CODE_OF_CONDUCT.md:1]: This should be in sync with [all other repos](https://github.com/search?q=org%3Aopensearch-project+https%3A%2F%2Faws.github.io%2Fcode-of-conduct-faq&type=code) OpenSearch has. Hence, `##` is correct.

**Review by AMoo-Miki — APPROVED** [review]

**Pull request merged**